Brewfile

```diff
@@ -0,0 +1,5 @@
+brew "git-lfs"
```

Comment:
We don't use this anywhere, and AFAICR we rewrote history to completely remove it from our repo, so we can kill it off in the Brewfile.